### PR TITLE
docs updated for CSI-PowerFlex driver v2.6.0

### PR DESCRIPTION
**Pull request by khareRajshree**
# Description
Updated release to 2.6.0
Updated feature list and release notes

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/583 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No edits are required.

**Review by VamsiSiddu-7 — APPROVED** [review]

**Review by ashleyvjoy — APPROVED** [review]

File: content/docs/csidriver/installation/operator/powerflex.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Kubernetes Operators make it easy to deploy and manage the entire lifecycle of c
   - Optionally, enable sdc monitor by uncommenting the section for sidecar in manifest yaml. Please note the following: 
     - **If using sidecar**, you will need to edit the value fields under the HOST_PID and MDM fields by filling the empty quotes with host PID and the MDM IPs. 
     - **If not using sidecar**, please leave this commented out -- otherwise, the empty fields will cause errors.
-##### Example CR:  [config/samples/vxflex_v250_ops_411.yaml](https://github.com/dell/dell-csi-operator/blob/master/samples/vxflex_v250_ops_411.yaml)
+##### Example CR:  [config/samples/vxflex_v260_ops_411.yaml](https://github.com/dell/dell-csi-operator/blob/master/samples/vxflex_v260_ops_411.yaml)
```
Comment (JacobGros) [inline content/docs/csidriver/installation/operator/powerflex.md:46]: this link works, but we can change master -> main branch in url.

File: content/docs/csidriver/upgradation/drivers/powerflex.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ Description: Upgrade PowerFlex CSI driver
 
 You can upgrade the CSI Driver for Dell PowerFlex using Helm or Dell CSI Operator.
 
-## Update Driver from v2.4 to v2.5 using Helm
+## Update Driver from v2.5 to v2.6 using Helm
 **Steps**
-1. Run `git clone -b v2.5.0 https://github.com/dell/csi-powerflex.git` to clone the git repository and get the v2.3.0 driver.
+1. Run `git clone -b v2.6.0 https://github.com/dell/csi-powerflex.git` to clone the git repository and get the v2.3.0 driver.
```
Comment (JacobGros) [inline content/docs/csidriver/upgradation/drivers/powerflex.md:15]: Should
"and get the v2.3.0 driver" be changed to 2.6.0?

**Review by bharathsreekanth — APPROVED** [review]
Could you also update the RHEL version to more exact numbers, like what I see for PowerStore.

File: content/docs/csidriver/upgradation/drivers/powerflex.md (hunk 2 above)
Comment (khareRajshree) [reply content/docs/csidriver/upgradation/drivers/powerflex.md]: updated

File: content/docs/csidriver/installation/operator/powerflex.md (hunk 1 above)
Comment (khareRajshree) [reply content/docs/csidriver/installation/operator/powerflex.md]: updated

**Comment by khareRajshree** [comment]
> Could you also update the RHEL version to more exact numbers, like what I see for PowerStore.

updated

**Review by JacobGros — APPROVED** [review]
LGTM

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this.  No edits are required.

**Pull request merged**